roles/servicetelemetry/templates/manifest_grafana_v5.j2

```diff
@@ -0,0 +1,95 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: {{ ansible_operator_meta.name }}-grafana
+  namespace: {{ ansible_operator_meta.namespace }}
+  labels:
+    dashboards: "stf"
+spec:
+  serviceAccount:
+    metadata:
+      annotations:
+        serviceaccounts.openshift.io/oauth-redirectreference.primary: '{{ grafana_oauth_redir_ref | to_json }}'
+{% if servicetelemetry_vars.graphing.grafana.ingress_enabled is defined and servicetelemetry_vars.graphing.grafana.ingress_enabled %}
+  route:
+    spec:
+      port:
+        targetPort: web
+      tls:
+        termination: reencrypt
+      to:
+        kind: Service
+        name: {{ ansible_operator_meta.name }}-grafana-service
+        weight: 100
+      wildcardPolicy: None
+{% endif %}
+  client:
+    preferIngress: false
+  config:
+    auth:
+      disable_signout_menu: "{{ servicetelemetry_vars.graphing.grafana.disable_signout_menu }}"
+      disable_login_form: "True"
+    auth.anonymous:
+      enabled: "True"
+    auth.proxy:
```

Comment:
I think this auth.proxy config could go away. I experimented with it in the last round of auth changes but ultimately found it unhelpful. The existing Grafana v4 config does not depend on any headers from the proxy, instead treating any user who makes it through the auth proxy as an anonymous admin.

Have you found any benefits to this config other than it showing your username in grafana? Does it treat the named users as admin?

Reply:
No benefit necessarily. As noted in an earlier comment, most of this is hear because I followed the example to get things working. I am going to attempt to drop this and make sure I can still get through to the Grafana instance.

If this works, in theory I could add a simple flag to use this or not, but without a strong need for it, that's probably just more overhead work that isn't necessary for an unsupported component.

I will drop this and if it works as it did before, I will call that a win and a completed experience.